components/apimgt/org.wso2.carbon.apimgt.gateway/src/main/java/org/wso2/carbon/apimgt/gateway/handlers/graphQL/analytics/package-info.java

```diff
@@ -0,0 +1 @@
+package org.wso2.carbon.apimgt.gateway.handlers.graphQL.analytics;
```

Comment:
Seems this file has been added by mistake. Let's remove this

Reply:
Fixed via https://github.com/wso2/carbon-apimgt/pull/12441/commits/fcd4966a4edf7516fe689a328717370308cc02e5